input/fsh/valuesets/au-erequesting-ranzcr-radiology-procedures.fsh

```diff
@@ -0,0 +1,523 @@
+* $sct#433032002 "Removal of gastrostomy tube using fluoroscopic guidance"
+* $sct#55612007 "Hepatic venography"
+* $sct#168884001 "Hysterosalpingogram"
+* $sct#433173004 "Insertion of permanent inferior vena cava filter using fluoroscopic guidance with contrast"
+* $sct#432079006 "Removal of inferior vena cava filter using fluoroscopic guidance with contrast"
+* $sct#32265006 "Intravenous pyelogram"
+* $sct#174304002 "Hydrostatic reduction of intussusception of rectum"
+* $sct#419245009 "Fluoroscopic micturating cystourethrography"
+* $sct#175988004 "Nephrostogram"
+* $sct#431410007 "Nephrostomy using fluoroscopic guidance"
+* $sct#251000087100 "Left nephrostomy using fluoroscopic guidance"
+* $sct#261000087102 "Right nephrostomy using fluoroscopic guidance"
+* $sct#701000087108 "Bilateral nephrostomy using fluoroscopic guidance"
+* $sct#431341009 "Removal of nephrostomy tube using fluoroscopic guidance"
+* $sct#84200003 "Angiography of pulmonary arteries"
+* $sct#765041007 "Angiography of left pulmonary artery"
+* $sct#765042000 "Angiography of right pulmonary artery"
+* $sct#79516005 "Angiography of renal artery"
+* $sct#431484009 "Percutaneous transluminal angioplasty of renal artery using fluoroscopic guidance with contrast"
+* $sct#331000087101 "Angioplasty of left renal artery using fluoroscopic guidance with contrast"
+* $sct#341000087107 "Angioplasty of right renal artery using fluoroscopic guidance with contrast"
+* $sct#751000087109 "Angioplasty of bilateral renal arteries using fluoroscopic guidance with contrast"
+* $sct#405250000 "Retrograde pyelogram"
+* $sct#1010454006 "Retrograde pyelography of left renal pelvis"
+* $sct#1010455007 "Retrograde pyelography of right renal pelvis"
+* $sct#241233006 "Angiography of spinal artery"
+* $sct#431298004 "Fluoroscopic myelogram of lumbar spine with cervical puncture"
+* $sct#241147001 "Tube esophagram"
+* $sct#703100007 "Ureteric stent procedure using fluoroscopic guidance"
+* $sct#168864000 "Urethrogram"
+* $sct#710294007 "Urodynamic study using fluoroscopic guidance"
+* $sct#703106001 "Vascular stent procedure using fluoroscopic guidance"
+* $sct#4970003 "Venography"
+* $sct#431911003 "Vertebroplasty using fluoroscopic guidance"
+* $sct#371576000 "Video imaging procedure"
+* $sct#241149003 "Videofluoroscopy swallow"
+* $sct#710132004 "Insertion of catheter into artery using fluoroscopic guidance with contrast"
+* $sct#432945008 "Percutaneous removal of calculus from biliary tract using fluoroscopic guidance"
+* $sct#431845006 "Replacement of biliary drain using fluoroscopic guidance"
+* $sct#1222890004 "Biliary drainage using imaging guidance"
+* $sct#83735008 "Insertion of biliary stent by endoscopic retrograde cholangiopancreatography"
+* $sct#241232001 "Angiography of bronchial artery"
+* $sct#85895003 "Contrast bronchogram"
+* $sct#438523006 "Percutaneous transluminal balloon test occlusion of middle cerebral artery using fluoroscopic guidance with contrast"
```

Comment:
@mjosborne1 Confirming these are the procedure names from SCT?

Reply:
I was lazy and used the FSNs - the semantic tag. I will adjust my script to query for the PT.

Reply:
Now using SNOMED CT PTs retreived from onto R4

Reply:
Can PTs be removed altogether? These manual inclusions may become outdated soon if SNOMED CT PTs get updated, which would then require updates of this VS.

Reply:
I have removed PTs with the latest update

Reply:
> I have removed PTs with the latest update

Confirmed.